containers/podman-desktop
# feat: send contexts status on different channels

### What does this PR do?

Use 4 different channels to send information to the frontend:

- status and number of a selected list of resources (pods, deployments) for all contexts (including the current one)
- Status and number of a selected list of resources (pods, deployments) for current context
- Pods in the current context
- Deploymenets in the current context

New state is sent or not into channels depending on the event (for example, the "Pods in current context" is not sent when a new Deployment is added to a context)

More unit tests 

### Screenshot / video of UI



### What issues does this PR fix or reference?

Fixes #6171

### How to test this PR?

Enable Kubernetes on Podman Desktop, start/stop deployments, start/stop clusters, and check data is always valid on Deployments List page and in Kubernetes Contexts page
